src/modules/General/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
```

Comment:
It's duplicated file same as DateTimePicker 
You can create a general component and pass a prop for handling `date_time | time | date`

Reply:
In fact they use different MUI components with different props. I think it is better not to merge them. However currently we don't need DateTimePicker, I prefer to keep it, maybe later we need it

Reply:
I know MUI component is different but because all things are same except the MUI component i prefer to use props instead of creating thousand same files 
Ok i merge this PR for now and i will handle it